example.tex

```diff
@@ -247,13 +247,13 @@ \subsubsection{Using \lstinline!themecolor!}
 \end{lstlisting}
 
 \begingroup
-\setthemecolor[PhbMauve]
+\DndSetThemeColor[PhbMauve]
```

Comment:
Don't forget the two calls above, as well as the references in the documentation.

Reply:
Done.